content/docs/user-guide/gems/reference/terrain/layer_spawner.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Layer Spawner component
+linktitle: LayerSpawner
+description: ' Open 3D Engine (O3DE) Terrain Layer Spawner reference. '
+weight: 100
+---
+
+{{< preview-migrated >}}
+
+The **Layer Spawner** component defines the bounds of a terrain layer, and enables the ordering of multiple layers.  
```

Comment:
Same as above, should mention that it actually spawns the region, it doesn't just define the bounds.